src/protocols/ssh/settings.h

```diff
@@ -223,6 +223,11 @@ typedef struct guac_ssh_settings {
      */
     int server_alive_interval;
 
+    /**
+     * The decismal ASCII code of the command to send for backspace.
```

Comment:
"decismal" is a typo, but that may be moot as this is technically incorrect. While the string from which this `int` was parsed may have been decimal, at this point it's just an `int`.